### DISCUSSION TITLE
Atom Not Showing Git Branch

### DISCUSSION THREAD

**Discussion by MaxTheSpy** [atom/atom#22998] [Q&A]
Good morning everyone. I use two computers to work on my programming. I use my home computer and work computer.

My work computer shows git branches Orign, RemoteDEV
My home computer shows git branches Origin, Dev

In a perfect world, I would be able to remove the RemoteDEV branch and just keep Origin and dev, but I cant seem to get my work computer to show dev, or my home computer to show Remotedev. meaning that I have to make a pull request in git and merge RemoteDEV to DEV in order to work from home. and if I go to work, I have to merge Dev, into RemoteDEV.

How to I show Origin (Main) and Dev on both computers?

**Comment by nazimboudeffa** [comment]
Hi,

I don't use a lot the branches feature when coding because I work mostly alone on my projects, just let them public but this question interests me as when I am in travel I like to use another computer to code

So basically what I do is that I clone the repo on my laptop, then I guess if I have some branches it just will switch to the branch by typing `git branch dev` for example or to get back to main by typing `git branch main`

In atom the branch will be shown automatically in the editor, I have tested that one time

**Reply by MaxTheSpy** [reply]
I guess I dont understand where you are typing in `git branch dev`

**Reply by icecream17** [reply]
cmd

(Edit: I'm assuming you're on windows. Press: <kbd>⊞ Win</kbd> then <kbd>c</kbd><kbd>m</kbd><kbd>d</kbd> then <kbd>Enter</kbd>. You might need to install git-for-windows. I'm not sure if the solution works.)

**Reply by nazimboudeffa** [reply]
I used to use [cmder](https://cmder.net/) but just cmd works on windows, otherwise, PowerShell, or as said @icecream17 git-for-windows

[A quick search on Google](https://stackoverflow.com/questions/43013195/how-to-open-the-terminal-in-atom) shows that there are special terminals for Atom (inside the editor) but it seems that they are not maintained since everyone have moved to VSCode maybe not all the community

Also have you tryed with Git or GitHub for Atom ?

**Reply by MaxTheSpy** [reply]
> I used to use [cmder](https://cmder.net/) but just cmd works on windows, otherwise, PowerShell, or as said @icecream17 git-for-windows
> 
> [A quick search on Google](https://stackoverflow.com/questions/43013195/how-to-open-the-terminal-in-atom) shows that there are special terminals for Atom (inside the editor) but it seems that they are not maintained since everyone have moved to VSCode maybe not all the community
> 
> Also have you tryed with Git or GitHub for Atom ?

I have Git installed on atom. and its only finding the 2 branches. Third is still missing. Will try installing Git for windows tonight and see if that works.
